examples/actix/main.rs

```diff
@@ -229,7 +229,7 @@ fn main() {
     let domain = opt.rp_id.clone();
 
     // Generate TLS certs as needed.
-    let ssl_params = generate_dyn_ssl_params(domain.as_str());
+    // let ssl_params = generate_dyn_ssl_params(domain.as_str());
```

Comment:
If you remove this then the localhost test server won't work because the browser won't send over non-https IIRC.

Reply:
From:

https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API

> Please note: both create() and get() require a Secure Context (e.g. - the server is connected by https or is the localhost), and will not be available for use if the browser is not operating in a secure context.

I verified it works for me on localhost:<port>. Might you try again and see if it fails for you? Maybe one browser allows it and another doesn't?

Reply:
I remember what it was, it was when I was testing from my ipad to my development machine, that's why it needed the https generator. What's the main reason to disable it?

Reply:
I got weird issues on safari when the certificate changed, it wouldn't let me skip pass the warning. in general, it's a bit harder to test with https locally (various browsers do different things here). We can meet in the middle and just make it an option at run-time, wdyt?

Reply:
Yep, and I'm happy for it to default to "off", so that only when I or others need some esoteric test they can enable it. Thanks!